docs/en/stack/ml/df-analytics/ml-supervised-workflow.asciidoc

```diff
@@ -0,0 +1,139 @@
+[role="xpack"]
+[[ml-supervised-workflow]]
+= Introduction to supervised learning
+
+Elastic supervised learning allows you to train a {ml} model based on training 
```

Comment:
I always prefer "enable" to "allow" since the former implies a sense of empowerment, whereas the latter makes me think of security/authority or permission to do something.
```suggestion
Elastic supervised learning enables you to train a {ml} model based on training 
```